lua/nvim-treesitter/textobjects/attach.lua

```diff
@@ -12,8 +12,13 @@ local function make_dot_repeatable(fn)
 end
 
 function M.make_attach(functions, submodule, keymap_modes, opts)
-  keymap_modes = keymap_modes or "n"
+  if type(keymap_modes) == "string" then
+    keymap_modes = { keymap_modes }
+  elseif type(keymap_modes) ~= "table" then
+    keymap_modes = { "n" }
+  end
   opts = opts or {}
+
```

Comment:
I tested with this fix but the issue persists. The error originates from `make_detach`.

Reply:
Did I make another mistake?

Reply:
I see, sorry maybe I need more sleep haha

Reply:
That's the right fix, just not the right location :sweat_smile:. `make_detach` is the origin of the error while you applied the fix for `make_attach`.

Reply:
I thought I changed the detach part but I was rushing. Sorry about that. Now with #367 it should be good right?